tests/integration/components/objective-sort-manager-test.js

```diff
@@ -34,8 +34,10 @@ module('Integration | Component | objective sort manager', function(hooks) {
     });
 
     this.set('subject', subject);
+    this.set('cancel', parseInt);
```

Comment:
Instead of this syntax consider `@cancel={{noopl}}` in the template which does the same thing, but to me is clearer.

Reply:
oh, that's great! handled.

Reply:
Still showing up as parseInt for me here. Push again?

Reply:
ah yes. missed a bunch of those. fixed now.